examples/chip-tool/commands/pairing/PairingCommand.h

```diff
@@ -202,6 +202,18 @@ class PairingCommand : public CHIPCommand,
             AddArgument("dst-offset", &mComplex_DSTOffsets,
                         "DSTOffset list to use when setting Time Synchronization cluster's DSTOffset attribute",
                         Argument::kOptional);
+
+            AddArgument("require-tc-acknowledgements", 0, 1, &mRequireTCAcknowledgements,
+                        "Terms and Conditions acknowledgements is known to be required or not, (when required, the commissioner "
+                        "will wait until they are provided)");
```

Comment:
This does not seem like the right description, in that there is no waiting; when required you can provide them via the other args, right?

Also descriptions for optional args should say what the default behavior is.

Reply:
I updated the description each of the three new arguments.